docs/conf.py

```diff
@@ -61,9 +61,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.4'
+version = '0.5'
```

Comment:
Can we extract this variable to somewhere? Easy place to miss. At least it must be added to the release checklist.